flight/PiOS/inc/pios_flight_config.h

```diff
@@ -0,0 +1,59 @@
+#ifndef _PIOS_FLIGHT_CONFIG_H
+#define _PIOS_FLIGHT_CONFIG_H
+
+// Top level PIOS Subsystems
+#define PIOS_INCLUDE_ADC
+#define PIOS_INCLUDE_ANNUNC
+#define PIOS_INCLUDE_BL_HELPER
+#define PIOS_INCLUDE_COM
+#define PIOS_INCLUDE_EXTI
+#define PIOS_INCLUDE_GPS
+#define PIOS_INCLUDE_GPS_NMEA_PARSER
+#define PIOS_INCLUDE_GPS_UBX_PARSER
+#define PIOS_INCLUDE_IAP
+#define PIOS_INCLUDE_INITCALL
+#define PIOS_INCLUDE_PPM
+#define PIOS_INCLUDE_RCVR
+#define PIOS_INCLUDE_RTC
+#define PIOS_INCLUDE_SERVO
+#define PIOS_INCLUDE_SYS
+#define PIOS_INCLUDE_TIM
+#define PIOS_INCLUDE_USART
+#define PIOS_INCLUDE_USB
+#define PIOS_INCLUDE_USB_CDC
+#define PIOS_INCLUDE_USB_HID
+#define PIOS_INCLUDE_WDG
+
+#define WDG_STATS_DIAGNOSTICS
+
+#define PIOS_INCLUDE_FLASH
+#define PIOS_INCLUDE_FLASH_INTERNAL
+#define PIOS_INCLUDE_LOGFS_SETTINGS
+
+#define PIOS_INCLUDE_CHIBIOS
+
+// Resource limits
+#define CPULOAD_LIMIT_WARNING		80
+#define CPULOAD_LIMIT_CRITICAL		95
+#define IRQSTACK_LIMIT_WARNING		150
+#define IRQSTACK_LIMIT_CRITICAL		80
```

Comment:
Tabs for alignment or variable-width stuff is a bad idea but I know this is copy-paste so meh. :wink: